widget/ui/src/containers/SwapInput/SwapInput.tsx

```diff
@@ -72,36 +78,43 @@ export function SwapInput(props: SwapInputProps) {
             </>
           ) : (
             <>
-              <InputAmount
-                disabled={props.disabled || props.mode === 'To'}
-                style={{ padding: 0 }}
-                value={props.price.value}
-                type={'onInputChange' in props ? 'number' : 'text'}
-                size="large"
-                placeholder="0"
-                variant="ghost"
-                min={0}
-                {...('onInputChange' in props && {
-                  onChange: (event: React.ChangeEvent<HTMLInputElement>) =>
-                    props.onInputChange(event.target.value),
-                })}
-              />
+              <Tooltip
+                content={props.price.realValue}
+                open={props.mode === 'From' ? false : undefined}>
+                <InputAmount
+                  disabled={props.disabled || props.mode === 'To'}
+                  style={{ padding: 0 }}
+                  value={props.price.value}
+                  type={'onInputChange' in props ? 'number' : 'text'}
+                  size="large"
+                  placeholder="0"
+                  variant="ghost"
+                  min={0}
+                  {...('onInputChange' in props && {
+                    onChange: (event: React.ChangeEvent<HTMLInputElement>) =>
+                      props.onInputChange(event.target.value),
+                  })}
+                />
+              </Tooltip>
               {'percentageChange' in props ? (
                 <PriceImpact
                   size="large"
                   outputUsdValue={props.price.usdValue}
+                  realOutputUsdValue={props.price.realUsdValue}
                   error={props.price.error}
                   percentageChange={props.percentageChange}
                   warningLevel={props.warningLevel}
                 />
               ) : (
-                <ValueTypography hasWarning={!!props.price.error}>
-                  <Typography variant="body" size="medium">
-                    {props.price.usdValue
-                      ? `~$${props.price.usdValue}`
-                      : props.price.error}
-                  </Typography>
-                </ValueTypography>
+                <Tooltip content={props.price.realUsdValue} side="right">
+                  <ValueTypography hasWarning={!!props.price.error}>
+                    <Typography variant="body" size="medium">
+                      {props.price.usdValue
+                        ? `~$${props.price.usdValue}`
+                        : props.price.error}
+                    </Typography>
+                  </ValueTypography>
+                </Tooltip>
```

Comment:
For consistency and improved readability of the token amount, it is recommended to position the tooltip at the bottom: 
![image](https://github.com/rango-exchange/rango-client/assets/120931880/a4617796-8418-4bd1-9f80-a3112ef1d425)
![image](https://github.com/rango-exchange/rango-client/assets/120931880/a77db9ed-cf61-49b3-8ffa-fc4b0dbbf215)
![image](https://github.com/rango-exchange/rango-client/assets/120931880/448cdab6-6304-47aa-9b10-ebdae2ca6612)

Reply:
fixed.